prog/vm/nexus.rb

```diff
@@ -225,6 +239,26 @@ def before_run
 
     host.sshable.cmd(command)
 
+    hop_wait_for_slice
+  end
+
+  label def wait_for_slice
+    unless vm.vm_host_slice.nil?
```

Comment:
```suggestion
    if vm.vm_host_slice
```

Unless the value can be `false`, it's better to use it directly as opposed to `nil?`